dask/dataframe/core.py

```diff
@@ -2844,6 +2844,74 @@ def drop(self, labels, axis=0, errors='raise'):
     def merge(self, right, how='inner', on=None, left_on=None, right_on=None,
               left_index=False, right_index=False, suffixes=('_x', '_y'),
               indicator=False, npartitions=None, shuffle=None):
+        """Merge the DataFrame with another DataFrame
+
+        This will merge the two datasets, either on the indices, a certain column
+        in each dataset or the index in one dataset and the column in another.
+
+        1. Joining on indices: In this case the divisions are
+        aligned using the function ``dask.dataframe.multi.align_partitions``.
+        Afterwards, each partition is merged with the pandas merge function.
+
+        2. Joining one on index and one on column:
+        In this case the divisions of dataframe
+        merged by index ($d_i$) are used to divide the column merged dataframe
+        ($d_c$) one using ``dask.dataframe.multi.rearrange_by_divisions``.
+        In this case the merged dataframe ($d_m$) has the exact same divisions
+        as ($d_i$). This can lead to issues if you merge multiple rows from
+        ($d_c$) to one row in ($d_i$).
+
+        3. Joining both on columns:
+        In this case a hash join is performed using
+        ``dask.dataframe.multi.hash_join``.
+
+        Parameters
+        ----------
+        right: Dask DataFrame
+        how : {'left', 'right', 'outer', 'inner'}, default: 'inner'
+            How to handle the operation of the two objects.
+            * left: use calling frame's index (or column if on is specified)
+            * right: use other frame's index
+            * outer: form union of calling frame's index (or column if on is
+              specified) with other frame's index, and sort it
+              lexicographically
+            * inner: form intersection of calling frame's index (or column if
+              on is specified) with other frame's index, preserving the order
+              of the calling's one
+        on : label or list
+            Column or index level names to join on. These must be found in both
+            DataFrames. If on is None and not merging on indexes then this
+            defaults to the intersection of the columns in both DataFrames.
+        left_on : label or list, or array-like
+            Column to join on in the left DataFrame. Other than in pandas
+            arrays and lists are only support if their length is 1.
+        right_on : label or list, or array-like
+            Column to join on in the right DataFrame. Other than in pandas
+            arrays and lists are only support if their length is 1.
+        left_index : boolean, default False
+            Use the index from the left DataFrame as the join key.
+        right_index : boolean, default False
+            Use the index from the right DataFrame as the join key.
+        suffixes : 2-length sequence (tuple, list, …)
```

Comment:
This ellipsis may be the thing causing the encoding issue.

You can maybe add `# -*- coding: utf-8 -*-` to the top of the file.

Reply:
Good spot. I always forget that there is an automatic replacement for it in some IDEs...
I added the encoding to the file.